webapp/config/packages/prometheus.yaml

```diff
@@ -0,0 +1,11 @@
+services:
+    prometheus_storage:
+      public: false
+      class: \Prometheus\Storage\InMemory
+      # class: \Prometheus\Storage\APC
```

Comment:
Since you seem to set and return the values in the same function, does using this commented out APC storage even make sense? Otherwise we might as well remove it.

Reply:
This is probably leftover from when I was trying to include some generic http level metrics in the response as well(things like p95/99 response times, request counts, etc). I think inmemory only lives as long as the single request, while apc lives as long as the process is alive(allowing it to aggregate information across requests).